action/protocol/account/transfer.go

```diff
@@ -75,12 +75,10 @@ func (p *Protocol) handleTransfer(ctx context.Context, act action.Action, sm pro
 		return nil, errors.Wrapf(err, "failed to decode recipient address %s", tsf.Recipient())
 	}
 	recipientAcct, err := accountutil.LoadAccount(sm, recipientAddr)
-	if !fCtx.TolerateLegacyAddress {
-		if err != nil {
-			return nil, errors.Wrapf(err, "failed to load address %s", tsf.Recipient())
-		}
+	if err != nil {
```

Comment:
I will assume that no error was returned before the `toleratelegacyaddress` fork.

Reply:
right, fullsync test proves that